torch/serialization.py

```diff
@@ -350,8 +365,13 @@ def _check_dill_version(pickle_module) -> None:
                 pickle_module.__version__
             ))
 
-def save(obj, f: Union[str, os.PathLike, BinaryIO, IO[bytes]],
-         pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL, _use_new_zipfile_serialization=True) -> None:
+def save(
+    obj: object,
+    f: Union[str, os.PathLike, BinaryIO, IO[bytes]],
```

Comment:
Hmm, can you define something like `FileLikeType` and use it throughout the file?